lib/opencensus/trace/config.rb

```diff
@@ -64,6 +73,14 @@ class << self
       #     an object with a call method that takes a single options hash.
       #     See OpenCensus::Trace::Samplers. The initial value is a Probability
       #     sampler with a default rate.
+      # *   `exporter` The exporter to use. Must be an exporter, an object with
+      #     an export method that takes an array of Span objects. See
+      #     OpenCensus::Trace::Exporters. The initial value is a Logger exporter
+      #     that logs to STDOUT.
+      # *   `formatter` The trace context propagation formatter to use. Must be
```

Comment:
Probably should specify that it is for outgoing HTTP requests (i.e. the binary formatter is not a legal value). In fact, we might consider naming it `http_formatter` to make that more explicit.